src/Application/Cli/TravisAutoDeployCommand.php

```diff
@@ -106,20 +99,17 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $this->commandRunner->run('git config --global user.name "${GIT_NAME}"');
         $this->commandRunner->run('git config --global user.email "${GIT_EMAIL}"');
 
-        // getting current php version to only deploy once
-        $currentPhpVersion = getenv('TRAVIS_PHP_VERSION');
-        if ($input->getOption('php-version') != $currentPhpVersion) {
-            $output->writeln('<comment>This version of the documentation is already deployed</comment>');
-
-            return;
-        }
-
         // Generate the website
         $this->generator->generate($repository, $output);
 
         $output->writeln('');
 
         // Deploy it
         $this->deployer->deploy($repository, $output, $repositoryUrl, $targetBranch);
+
+        // generate couscous.phar to be uploaded when a new release is done
+        $this->commandRunner->run('bin/compile');
+        $output->writeln('<comment>PHAR file generated</comment>');
+
```

Comment:
I'm not sure I understand this -> this will be executed for users of Couscous, that script will not exist